src/app/(test)/yj/_components/ProductDetail.module.scss

```
@@ -0,0 +1,152 @@
+.detail-container {
+  display: flex;
+  gap: 7.2rem;
+  padding-bottom: 18rem;
+}
+
+.right-section {
+  width: 40%;
+}
+
+.name-button-section {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 0.8rem;
+
+  h1 {
+    @include pretendard-20-500;
+  }
+}
+
+.rate-section {
+  display: flex;
+  align-items: center;
+  gap: 1.2rem;
+  margin-bottom: 2rem;
+
+  h4 {
+    @include pretendard-14-400;
+
+    color: $gray-50;
+  }
+}
+
+.price {
+  @include pretendard-24-700;
+
+  margin-bottom: 2rem;
+}
+
+@mixin box {
+  padding: 2rem 0;
+  border-top: 0.1rem solid $gray-10;
+}
+
+.explain-text {
+  @include pretendard-16-400;
+
+  padding-bottom: 0.8rem;
+
+  span {
+    @include pretendard-16-500;
+
+    padding-right: 2rem;
+  }
+}
+
+.explain-title {
+  @include pretendard-20-500;
+
+  margin-bottom: 2rem;
+}
+
+.delivery-section {
+  @include box;
+}
+
+.point-section {
+  @include box;
+}
+
+.option-section {
+  @include box;
+}
+
```

아마도,,? 저의 기억상으로는 아래처럼 써도 됐던걸로 기억합니동,,

```suggestion
.delivery-section, .point-section, .option-section {
  @include box;
}
```